apps/Gruntfile.js

```diff
@@ -801,6 +801,8 @@ describe('entry tests', () => {
       './src/sites/code.org/pages/views/share_privacy.js',
     'code.org/views/theme_common_head_after':
       './src/sites/code.org/pages/views/theme_common_head_after.js',
+    'code.org/views/theme_google_analytics':
+      './src/sites/code.org/pages/views/theme_google_analytics.js',
```

Comment:
Were there situations/pages where the existing JS outputs weren't enough?

Reply:
I don't think we had a single output containing the initialization of google Universal or setting up dimension, events or cookies before. It used to be in an ebr file 
and the other sites were doing their own initialization and code duplication. 

Also keep in mind this is the first time I am doing this^

I decided to crete its own theme so it can be reused and inject it when/where needed